Description: Use standard libdir folders
 Do not postfix 64 in install libdir, use multiarch triplet
Author: arnebe <arne@alamut.de> 
Forwarded: no
Last-Update: 2012-05-18

--- a/cmake_modules/FindOS.cmake
+++ b/cmake_modules/FindOS.cmake
@@ -46,14 +46,3 @@
 
 MESSAGE (STATUS "Got System Processor ${CMAKE_SYSTEM_PROCESSOR}")
 
-# 32 or 64 bit Linux
-IF (PROJECT_OS_LINUX)
-    # Set the library directory suffix accordingly
-    IF (${CMAKE_SYSTEM_PROCESSOR} STREQUAL "x86_64")
-        SET (PROJECT_PROC_64BIT TRUE BOOL INTERNAL)
-        MESSAGE (STATUS "Linux x86_64 Detected")
-    ELSEIF (${CMAKE_SYSTEM_PROCESSOR} STREQUAL "ppc64")
-        MESSAGE (STATUS "Linux ppc64 Detected")
-        SET (PROJECT_PROC_64BIT TRUE BOOL INTERNAL)
-    ENDIF (${CMAKE_SYSTEM_PROCESSOR} STREQUAL "x86_64")
-ENDIF (PROJECT_OS_LINUX)
--- a/cmake_modules/SetupDirectories.cmake
+++ b/cmake_modules/SetupDirectories.cmake
@@ -11,13 +11,9 @@
 STRING (TOLOWER ${PROJECT_NAME} projectNameLower)
 SET (PROJECT_INCLUDE_INSTALL_DIR "include/${projectNameLower}")
 
-IF (PROJECT_PROC_64BIT)
-    SET (LIB_SUFFIX "64" CACHE STRING "Suffix for library installation directory")
-ELSE (PROJECT_PROC_64BIT)
-    SET (LIB_SUFFIX "" CACHE STRING "Suffix for library installation directory")
-ENDIF (PROJECT_PROC_64BIT)
-
-SET (PROJECT_LIBRARY_INSTALL_DIR "lib${LIB_SUFFIX}")
+IF (NOT DEFINED PROJECT_LIBRARY_INSTALL_DIR)
+	SET (PROJECT_LIBRARY_INSTALL_DIR "lib" CACHE PATH "object code libraries")
+ENDIF ()
 
 MESSAGE (STATUS "Headers will be installed to ${CMAKE_INSTALL_PREFIX}/${PROJECT_INCLUDE_INSTALL_DIR}")
 MESSAGE (STATUS "Libraries will be installed to ${CMAKE_INSTALL_PREFIX}/${PROJECT_LIBRARY_INSTALL_DIR}")
